sources/packages/web/src/assets/css/base.scss

```diff
@@ -283,3 +284,25 @@ html * {
 .modal-height .v-card-title {
   min-height: 50px;
 }
+
+.v-table.bordered {
+  border: 1px solid $common-border-color;
+}
+
+.tooltip-icon {
+  color: $color-muted;
+  font-size: 16px !important;
```

Comment:
Pls use global variables. Same for line 295,304,305,306

Reply:
Line 305,306 does not fall under a common property template to use from global variable.

![image](https://user-images.githubusercontent.com/54600590/191578305-0ebfb43e-385d-4458-9f66-0b6afee7353a.png)

Reply:
in that case, pls double-check with @hellolynn-tbtb. if it's the really font size or line size.

Reply:
Each icons are internally fonts and font-size is directly proportional to the icon size. I can check with @hellolynn-tbtb on all the properties and see if they follow a pattern to create global variables.

Reply:
actually, if it's for the size of the icon you can use the `size` attribute of the icon. it converts to `px` if you want `vh` or `%` you can do that too

Reply:
Yes. Just spoke to @hellolynn-tbtb and decided that for icons we are using only size attribute. font-weight does not matter.
Hence I will be removing this class status-info-icon.